index.js

```diff
@@ -98,15 +93,27 @@ var Frame = React.createClass({
       setTimeout(this.renderFrameContents, 0);
     }
   },
+  getMountTarget: function() {
+    var doc = this.getDoc();
+
+    if(this.props.mountTarget) {
+      return doc.querySelector(this.props.mountTarget);
+    }
+
+    return doc.body.children[0];
+  },
+  getDoc: function() {
+    return ReactDOM.findDOMNode(this).contentDocument;
```

Comment:
Since you're touching this part of the code we may as well move getting the DOM node over to a ref since `findDOMNode` will be deprecated eventually.

```html
<iframe ref={function(frame) { return this._frame = frame; }} />
```

Then you can just use `this._frame` to grab the iframe document.